tests/test_hf_api.py

```diff
@@ -1248,3 +1248,11 @@ def test_repo_type_and_id_from_hf_id(self):
 
         for key, value in possible_values.items():
             self.assertEqual(repo_type_and_id_from_hf_id(key), tuple(value))
+
+
+def test_delete_repo_error_message():
+    # test for #751
+    api = HfApi(endpoint=ENDPOINT_STAGING)
+    token = api.login(username=USER, password=PASS)
+    with pytest.raises(HTTPError, match="Conflict"):
+        api.delete_repo("repo-that-does-not-exist", token=token)
```

Comment:
Generally we try to put all tests in a test overriding `unittest.TestCase`. 

Is it possible to do the same for this test?